interop-cats/shared/src/main/scala/zio/interop/Queue.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017-2019 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.interop
+
+import cats.effect.LiftIO
+import zio.{ Queue => ZioQueue, Runtime, UIO, ZEnv }
+
+object Queue {
+
+  /**
+   * @see [[ZioQueue.bounded]]
+   */
+  final def bounded[F[+_], R, A](capacity: Int)(implicit R: Runtime[ZEnv], F: LiftIO[F]): F[Queue[F, A]] =
+    create(ZioQueue.bounded[A](capacity))
+
+  /**
+   * @see [[ZioQueue.dropping]]
```

Comment:
Using `[[...]]` makes Scaladoc finds the actual type I think? Isn't it going to fail generating the doc? (I'm not sure)

Reply:
Ah I think you're right. Looks like we're not doing the documentation as part of CI for this project. Probably best to do it the same way we did for Schedule I would think.

Reply:
I actually ran sbt doc against this and it compiled but I can revert to the way Schedule does it

Reply:
I actually ran sbt doc against this and it compiled but I can revert to the way Schedule does it

Reply:
I actually ran sbt doc against this and it compiled but I can revert to the way Schedule does it

Reply:
Maybe look at the generated Scaladoc for this and Schedule and see if the links work?

Reply:
Will do!

Reply:
Seems like it was complaining about these and it wasn't for the others so I shifted to the way Schedule does it